Taskfile.yml

```diff
@@ -733,6 +734,71 @@ tasks:
       WEBHOOK_OPTIONS: webhook output:webhook:artifacts:config={{.CONTROLLER_OUTPUT}}/webhook
       RBAC_OPTIONS: rbac:roleName=manager-role output:rbac:artifacts:config={{.CONTROLLER_OUTPUT}}/rbac
 
+  controller:check-crd-size:
+    desc: Check that generated CRDs are not too large for Kubernetes
+    deps:
+      - controller:generate-crds
+    dir: "{{.CONTROLLER_ROOT}}"
+    silent: true
+    sources:
+      - "{{.CONTROLLER_OUTPUT}}/crd/generated/bases/*.yaml"
+    cmds:
```

Comment:
minor: Should we put this in the scripts folder and call the script here, rather than inlining it in the taskfile?

Obviously would still have this taskfile entry but just call the script.